src/ServiceManagerBase.sol

```diff
@@ -69,4 +76,40 @@ contract ServiceManagerBase is IServiceManager, OwnableUpgradeable {
     function deregisterOperatorFromAVS(address operator) public virtual onlyRegistryCoordinator {
         delegationManager.deregisterOperatorFromAVS(operator);
     }
+
+    /**
+     * @notice Returns the list of strategies that the operator has potentially restaked on the AVS
+     * @param operator The address of the operator to get restaked strategies for
+     * @dev This function is intended to be called off-chain
+     * @dev No guarantee is made on whether the operator has shares for a strategy in a quorum or uniqueness 
+     *      of each element in the returned array. The off-chain service should do that validation separately
+     */
+    function getOperatorRestakedStrategies(address operator) external view returns (address[] memory) {
+        bytes32 operatorId = registryCoordinator.getOperatorId(operator);
+        uint192 operatorBitmap = registryCoordinator.getCurrentQuorumBitmap(operatorId);
+
+        if (operatorBitmap == 0 || registryCoordinator.quorumCount() == 0) {
+            return new address[](0);
+        }
+
+        // Get number of strategies for each quorum in operator bitmap
+        bytes memory operatorRestakedQuorums = BitmapUtils.bitmapToBytesArray(operatorBitmap);
+        uint256 strategyCount;
+        for(uint256 i = 0; i < operatorRestakedQuorums.length; i++) {
+            strategyCount += stakeRegistry.strategyParamsLength(uint8(operatorRestakedQuorums[i]));
```

Comment:
will be an overestimate if there are duplicates

Reply:
Yup, noted in dev comment

Reply:
I don't think this is an estimate so much as a specific count. If there are duplicates, the return array will have duplicates

I don't think that's a problem given it's consumed by offchain code though